docs/docs/Contributing/contributing-telemetry.mdx

```diff
@@ -3,47 +3,66 @@ title: Telemetry
 
 - **IsWebhook**: Indicates whether the operation was triggered with a webhook.
 - **Seconds**: Duration in seconds for how long the operation lasted, providing insights into performance.
 - **Success**: Boolean value indicating whether the operation was successful, helping identify potential errors or issues.
 - **ErrorMessage**: Provides error message details if the operation was unsuccessful, aiding in troubleshooting and enhancements.
 
-### Shutdown {#081e4bd4faec430fb05b657026d1a69c}
+### Shutdown
+
+This telemetry event captures information about application lifecycle and runtime duration.
 
-- **Time Running**: Total runtime before shutdown, useful for understanding application lifecycle and optimizing uptime.
+- **Time Running**: Total runtime before shutdown, which is useful for understanding the application lifecycle and optimizing uptime.
 
-### Version {#dc09f6aba6c64c7b8dad3d86a7cba6d6}
+### Version
+
+This telemetry event is sent once when the telemetry service starts.
 
 - **Version**: The specific version of Langflow used, which helps in tracking feature adoption and compatibility.
-- **Platform**: Operating system of the host machine, which aids in focusing our support for popular platforms like Windows, macOS, and Linux.
+- **Platform**: Operating system of the host machine, which helps determine the most popular platforms for development and testing efforts.
 - **Python**: The version of Python used, assisting in maintaining compatibility and support for various Python versions.
-- **Arch**: Architecture of the system (e.g., x86, ARM), which helps optimize our software for different hardware.
+- **Arch**: Architecture of the system, such as x86 or ARM, which helps prioritize hardware optimization and testing in the Langflow codebase.
 - **AutoLogin**: Indicates whether the auto-login feature is enabled, reflecting user preference settings.
 - **CacheType**: Type of caching mechanism used, which impacts performance and efficiency.
-- **BackendOnly**: Boolean indicating whether you are running Langflow in a backend-only mode, useful for understanding deployment configurations.
+- **BackendOnly**: Boolean indicating whether Langflow is running in backend-only mode, useful for understanding deployment configurations.
+- **Desktop**: Indicates whether Langflow is running in desktop mode (Langflow Desktop), helping to understand usage patterns across different deployment types.
+
+### Playground
 
-### Playground {#ae6c3859f612441db3c15a7155e9f920}
+This telemetry event monitors performance and usage patterns in the Playground environment.
 
 - **Seconds**: Duration in seconds for Playground execution, offering insights into performance during testing or experimental stages.
 - **ComponentCount**: Number of components used in the Playground, which helps understand complexity and usage patterns.
 - **Success**: Success status of the Playground operation, aiding in identifying the stability of experimental features.
```

Comment:
```suggestion
This telemetry event monitors performance and usage patterns in the **Playground** environment.

- **Seconds**: Duration in seconds for **Playground** execution, offering insights into performance during testing or experimental stages.
- **ComponentCount**: Number of components used in the **Playground**, which helps understand complexity and usage patterns.
- **Success**: Success status of the **Playground** operation, aiding in identifying the stability of experimental features.
```

Decided in LFOSS-1534 that we should always use `**Playground**` because it is the specific name/screen in the UI, not a generic concept.